src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -149,6 +149,8 @@ public string WarningsNotAsErrors
             get { return (string)_store[nameof(WarningsNotAsErrors)]; }
         }
 
+        public string NullableContextOptions { get { return null; } set { } }
```

Comment:
What is this needed for?

Reply:
It shouldn't be needed with 854772f94bd5a44aaf8f1bfcc00632e9209bb523, but I got a failure in CI anyway. I'm planning to submit a follow-up PR to revert this change and it will merge as soon as we can get it to pass, which at the latest will be whenever the change that removed this property ships in a preview image.

When the failure occurs, csproj projects fail to load correctly in integration test runs.

Reply:
➡️ Submitted #35725 to revert this change.